docs/cookbooks/ids/README.md

```diff
@@ -0,0 +1,173 @@
+Pods. In this scenario, Antrea is used for the default network. For the sake of
+this guide, we will use [Suricata](https://suricata.io/) as the threat detection
+engine, but similar steps should apply for other engines as well.
+
+The solution works by configuring a TrafficControl resource applying to specific
+Pods. Traffic originating from the Pods or destined to the Pods is mirrored, and
+then inspected by Suricata to provide threat detection.
+
+<!-- toc -->
+- [Prerequisites](#prerequisites)
+- [Practical steps](#practical-steps)
+  - [Step 1: Deploy Antrea](#step-1-deploy-antrea)
+  - [Step 2: Configure TrafficControl resource](#step-2-configure-trafficcontrol-resource)
+  - [Step 3: Deploy Suricata as a DaemonSet](#step-3-deploy-suricata-as-a-daemonset)
+- [Testing](#testing)
+<!-- /toc -->
+
+## Prerequisites
+
+The general prerequisites are:
+
+* a K8s cluster running a K8s version supported by Antrea.
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+The [TrafficControl](../../traffic-control.md) capability was added in Antrea
+version 1.7. Therefore, an Antrea version >= 1.7.0 should be used to configure
+Pod traffic mirroring.
+
+All the required software will be deployed using YAML manifests, and the
+corresponding container images will be downloaded from public registries.
+
+## Practical steps
+
+### Step 1: Deploy Antrea
+
+For detailed information on the Antrea requirements and instructions on how to
+deploy Antrea, please refer to [getting-started.md](../../getting-started.md).
+As of now, the `TrafficControl` feature gate is disabled by default, you will
+need to enable it like the following command.
+
+To deploy the latest version of Antrea, use:
+
+```bash
+curl -s https://raw.githubusercontent.com/antrea-io/antrea/main/build/yamls/antrea.yml | \
+  sed "s/.*TrafficControl:.*/      TrafficControl: true/" | \
+  kubectl apply -f -
+```
+
+You may also choose a [released Antrea
+version](https://github.com/antrea-io/antrea/releases).
+
+### Step 2: Configure TrafficControl resource
+
+To replicate Pod traffic to threat detection engines for analysis, create a
+TrafficControl with the `Mirror` action, and set the `targetPort` to an OVS
+internal port that threat detection engines will capture traffic from. This
+cookbook uses `tap0` as the port name and performs intrusion detection for Pods
+with the `app=web` label:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: crd.antrea.io/v1alpha2
+kind: TrafficControl
+metadata:
+  name: mirror-web-app-to-tap0
+spec:
+  appliedTo:
+    podSelector:
+      matchLabels:
+        app: web
+  direction: Both
+  action: Mirror
+  targetPort:
+    ovsInternal:
+      name: tap0
+EOF
+```
+
+### Step 3: Deploy Suricata as a DaemonSet
+
+Suricata supports many possible configuration options, we will focus on the
+basics in the cookbook. The YAML file for Suricata DaemonSet is included in the
```

Comment:
We should mention are using the image from https://github.com/jasonish/docker-suricata?